src/calculator/service.ts

```diff
@@ -47,7 +47,7 @@ function responseToTweet(response: any) : Tweet {
   return {
     text: {
       text: response.text,
-      lang: response.lang,
+      lang: ['en', 'es', 'fr'].includes(response.lang) ? response.lang : 'en',
```

Comment:
I would have put the languages in an array outside this line because if you add 50 more then it starts looking bad but for now, it looks fine